pages/empleo/index.vue

```diff
@@ -27,6 +27,13 @@ useHead({
             <p class="text-lg">
                 Encuentra ofertas de empleo e investigación relacionadas con el Procesamiento del Lenguaje Natural 🚀
             </p>
+            <div class="justify-center items-center mx-65">
+                <IconButtonLink
+                    :url='`https://twitter.com/intent/tweet?text=Acabamos de publicar una nueva oferta en la bolsa de empleo de @SomosNLP_ ! \n\n https://somosnlp.org/empleo`'>
```

Comment:
Sé que escribí yo la frase en el issue, esta me parece más directa, qué opinas?

```suggestion
                    :url='`https://twitter.com/intent/tweet?text=¡Nuevas ofertas en la bolsa de empleo de @SomosNLP_! \n\n\n Descúbrelas aquí: https://somosnlp.org/empleo`'>
```

Reply:
Esta mejor!, ya lo cambio.